Dockerfile-index

```diff
@@ -0,0 +1,49 @@
+# Process for indexing papers by arXiv ID.
+#
+# Expects a path to a list of newline-delimited arXiv IDs (versionless).
+#
+# Example (local stack):
+#
+#    $ mkdir /tmp/to_index
+#    $ cp arxiv_id_dump.txt /tmp/to_index
+#    $ docker run -it --network=arxivsearch_es_stack \
+#    >   -v /tmp/to_index:/to_index \
+#    >   -e ELASTICSEARCH_HOST=elasticsearch \
+#    >   arxiv/search-index /to_index/arxiv_id_dump.txt
+#
+# See also ELASTICSEARCH_* and METADATA_ENDPOINT parameters, below.
+
+FROM arxiv/base
+
+# Add Python consumer and configuration.
+ADD requirements.txt /opt/arxiv/
+ADD app.py /opt/arxiv/
+RUN pip install -U pip
+RUN pip install -r /opt/arxiv/requirements.txt
+
+ENV PATH "/opt/arxiv:${PATH}"
+
+ADD schema /opt/arxiv/schema
+ADD mappings /opt/arxiv/mappings
+ADD search /opt/arxiv/search
+ADD tests /opt/arxiv/tests
+ADD populate_test_metadata.py /opt/arxiv/
+
+WORKDIR /opt/arxiv/
+
+ENV LC_ALL en_US.utf8
+ENV LANG en_US.utf8
+ENV LOGLEVEL 10
+ENV FLASK_DEBUG 1
+ENV FLASK_APP /opt/arxiv/app.py
+
+ENV ELASTICSEARCH_HOST 127.0.0.1
+ENV ELASTICSEARCH_SCHEME http
+ENV ELASTICSEARCH_PORT 9200
+ENV ELASTICSEARCH_USER elastic
+ENV ELASTICSEARCH_PASSWORD changeme
+ENV METADATA_ENDPOINT https://arxiv.org/docmeta/
```

Comment:
At some point it might also be good to consider supporting more than one metadata endpoint, e.g.

``
ENV METADATA_ENDPOINT https://arxiv-web1.library.cornell.edu;https://arxiv-web2.library.cornell.edu;https://arxiv-web3.library.cornell.edu;https://arxiv-web4.library.cornell.edu
``
(then adding logic to choose one of the endpoints at metadata retrieval time)
The load balancer tends to be "sticky", so having the consumers distribute requests evenly across all the endpoints might yield a performance improvement. The DB and SFS on the classic system could end up being bottlenecks, so the improvement might only be marginal.

Reply:
Ok, sounds good. In this case I did the multi-endpoint routing outside the Dockerfile (see README.md) thinking that it would be more expedient